pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -183,6 +183,7 @@ type BundleDeploymentOptions struct {
 	ForceSyncGeneration int64             `json:"forceSyncGeneration,omitempty"`
 	YAML                *YAMLOptions      `json:"yaml,omitempty"`
 	Diff                *DiffOptions      `json:"diff,omitempty"`
+	Decrypt             bool              `json:"decrypt,omitempty"`
```

Comment:
We should use `*bool` since we want to know when it's unset. If `*bool` is `nil`, then we set the value to `true` to maintain existing behavior..

Reply:
Actually, if we create a `GitRepo` field of `*bool` type, this can actually be `bool` since we will set its value based on the former.

Reply:
agree, it should be type *bool. so
```
if decrypt == nil || *decrypt {
  decrypt()
}
```

Reply:
The only case where it doesn't need to be `*bool` though is if GitRepo is `*bool`. Theoretically, we could evaluate the GitRepo field and make this field a `bool`. Or, we could make this field `*bool` as well and copy the GitRepo field value directly to this one.

 I am fine either way.

Reply:
awesome, apologies I missed the message, just pushed the change to handle nil and make it a pointer, hope it's alright.